crates/ruff_formatter/src/builders.rs

```diff
@@ -841,23 +841,23 @@ impl<Context> std::fmt::Debug for Dedent<'_, Context> {
 ///
 /// # fn main() -> FormatResult<()> {
 /// let block = format!(SimpleFormatContext::default(), [
-///     text("root"),
+///     token("root"),
 ///     indent(&format_args![
 ///         hard_line_break(),
-///         text("indent level 1"),
+///         token("indent level 1"),
```

Comment:
This is arguably worse because these aren't really tokens but having a `text` builder just for the examples feels overkill (and the examples don't violate any constraint)